### PR TITLE
Multilinguism on interaactionScene

### PR DESCRIPTION
**Pull request by PillouMatou**
Now you can choose a language in the settings and it will be apply on all the application.

### CODE REVIEW

**Review by Noars — APPROVED** [review]
I tried to change the language several times and then refresh the page. No problem, the language we selected is well registered in the database.

**Review by Noars — APPROVED** [review]
I looked everywhere in the site with the language in French then in English and all the text fields are well translated according to the chosen language.

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "Hide": "Cacher",
+    "Remove": "Retirer",
+    "Rename": "Renommer",
+    "Export": "Exporter",
+    "Import": "Importer",
+    "Erase": "Gommer",
+    "Clear": "Effacer"
+  },
+  "Settings": {
+    "DwelltimeEnabled": "Activer la temporisation",
```
Comment (kuhlkrein) [inline src/assets/i18n/fr.json:22]: replace "temporisation" par "temps de fixation"

File: src/assets/i18n/fr.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "DwellTime": "Temps de temporisation",
+    "Back": "Retour",
+    "Selectalanguage": "Selectionne un langage",
```
Comment (kuhlkrein) [inline src/assets/i18n/fr.json:25]: "une langue" instead of "un langage"

File: src/assets/i18n/fr.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "ManageScenes": "Gérer les Scènes",
+    "Draw": "Dessiner",
+    "Settings": "Options",
+    "Scene": "Scène",
+    "Image": "Image",
+    "AddHotspot": "Ajouter un Hotspot",
+    "ModifyHotspot": "Modifier un Hotspot",
+    "DeleteHotspot": "Supprimer un Hotspot",
+    "Hide": "Cacher",
+    "Remove": "Retirer",
```
Comment (kuhlkrein) [inline src/assets/i18n/fr.json:14]: "Supprimer"

**Review by kuhlkrein — CHANGES_REQUESTED** [review]
Also all the "hotspot creation" par is not translated (the little dialog that pops when creating an hotspot)

**Comment by kuhlkrein** [comment]
Tests do not pass

**Comment by kuhlkrein** [comment]
Code works fine but I have an issue with translate:
image

**Pull request merged**